src/plays/wordle/Wordle.tsx

```diff
@@ -71,6 +71,7 @@ function Wordle(props: any): JSX.Element {
     // This method removes all children from element node
     errorSlideRef.current.replaceChildren();
     setStats(null);
+    setGameOver(false);
```

Comment:
@SamirMishra27 , why we are setting gameOver as false, could you please provide more details?

Reply:
`gameOver` is a state which indicates if the current game is over or not.
It's set to `true` after a game is finished.

If it is kept like that, then any unexpected keyboard key entries will be ignored here.
```ts


  function handleGameAction(action: WordleAction) {
    if (!action || gameOver) return;


    // Check and evaluate the guess
    if (action === 'Enter') {
```
So it's necessary to set it back to false then `reset()`'ing the data of all states when user starts a new game.